packages/vsce/src/utils/commandUtils.ts

```diff
@@ -73,3 +73,37 @@ export function splitCmciErrorMessage(message: any) {
 export function toArray<T>(input: T | T[]): T[] {
   return Array.isArray(input) ? input : [input];
 }
+
+export async function getResourceTree<T extends IResource>(
+  treeview: TreeView<any>,
+  nodes: any[],
+  targetResourceName: string
+): Promise<CICSResourceContainerNode<T> | undefined> {
+  let regionName = nodes[0].description?.toString() ?? "";
+
+  if (regionName.length > 0) {
+    regionName = regionName.match(/\(([^)]*)\)/)?.[1]?.trim() ?? regionName;
+
+    const regionsNode = nodes[0]
+      .getParent()
+      .getParent()
+      .children.find((ch: any) => ch.label.toString().includes("Regions")) as CICSRegionsContainer;
+
+    if (!regionsNode) return;
+
+    await treeview.reveal(regionsNode, { expand: true });
+
+    const regionTree = regionsNode.children.find((ch: any) => ch.label === regionName);
+    if (!regionTree) return;
```

Comment:
We get lint warnings for these and we tend to stick to bracketting _if_ statements.
```suggestion
    if (!regionsNode) {
      return;
    }

    await treeview.reveal(regionsNode, { expand: true });

    const regionTree = regionsNode.children.find((ch: any) => ch.label === regionName);
    if (!regionTree) {
      return;
    }
```

Reply:
```
packages/vsce/src/utils/commandUtils.ts
  92:23  warning  Expected { after 'if' condition  curly
  97:22  warning  Expected { after 'if' condition  curly
```